src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -96,8 +96,8 @@ private static void WriteClientFields(CodeWriter writer, LowLevelClient client)
                 writer.WriteFieldDeclaration(field);
             }
 
+            writer.Line();
```

Comment:
?

Reply:
Added this to add an empty line before the HttpPineline field. [Here](https://github.com/Azure/autorest.csharp/pull/1959/files#diff-0898eb7a22b4bff2482529f6c3099d51aa3268edfb3db4c42c079ed6fb617044) is the generated result.